source/change-streams/tests/change-streams-errors.yml

```diff
@@ -71,5 +71,4 @@ tests:
             z: 3
     result:
       error:
-        code: 280
-        errorLabels: [ "NonResumableChangeStreamError" ]
+        code: 20
```

Comment:
Let's also test for `280` error code on 4.2 servers.

Are we sure we don't want to push back on this error code change in the server?  Seems like this could break applications using older drivers against a 4.4 server.